BlockSettleSigner/signer.qrc

```diff
@@ -81,5 +81,16 @@
         <file>qml/BsHw/HwAvailableDevices.qml</file>
         <file>qml/BsHw/PassphraseDialog.qml</file>
         <file>qml/InfoBar.qml</file>
+        <file>bip39Dictionaries/chinese_simplified.txt</file>
+        <file>bip39Dictionaries/chinese_traditional.txt</file>
+        <file>bip39Dictionaries/czech.txt</file>
+        <file>bip39Dictionaries/english.txt</file>
+        <file>bip39Dictionaries/french.txt</file>
+        <file>bip39Dictionaries/italian.txt</file>
+        <file>bip39Dictionaries/japanese.txt</file>
+        <file>bip39Dictionaries/korean.txt</file>
+        <file>bip39Dictionaries/spanish.txt</file>
+        <file>qml/BsControls/BSPib39Input.qml</file>
```

Comment:
Let's not mix dictionaries and qml files - at least don't put them interleaved.

Reply:
Added a different qrc file for storing dictionaries there, thank you